README.md

```diff
@@ -116,16 +116,16 @@ and
 
 ## Using TokenBuffer API for Custom Tokenizers
 We offer a `TokenBuffer` API and supporting utility lexers
-for high speed tokenization.
+for high-speed tokenization.
 
 #### Writing your own TokenBuffer tokenizers
 
 `TokenBuffer` turns a string into a readable stream, used for building tokenizers.
-Utility lexers such as `spaces` and `number` read characters from the
+Utility lexers such as `spaces` and `numbers` read characters from the
```

Comment:
```suggestion
Utility lexers such as `spaces` and `number` read characters from the
```

`number` is the name of the function that reads out 1 number

It may not be a great name but it is thbe name we have